pandas/core/tools/timedeltas.py

```diff
@@ -16,31 +16,42 @@
 
 def to_timedelta(arg, unit='ns', box=True, errors='raise'):
     """
-    Convert argument to timedelta
+    Convert argument to timedelta.
+
+    Timedeltas are absolute differences in times, expressed in difference
+    units (e.g. days, hours, minutes, seconds). This method converts
+    an argument from a recognized timedelta format / value into
+    a Timedelta type.
 
     Parameters
     ----------
-    arg : string, timedelta, list, tuple, 1-d array, or Series
-    unit : str, optional
-        Denote the unit of the input, if input is an integer. Default 'ns'.
-        Possible values:
-        {'Y', 'M', 'W', 'D', 'days', 'day', 'hours', hour', 'hr', 'h',
-        'm', 'minute', 'min', 'minutes', 'T', 'S', 'seconds', 'sec', 'second',
-        'ms', 'milliseconds', 'millisecond', 'milli', 'millis', 'L',
-        'us', 'microseconds', 'microsecond', 'micro', 'micros', 'U',
-        'ns', 'nanoseconds', 'nano', 'nanos', 'nanosecond', 'N'}
-    box : boolean, default True
-        - If True returns a Timedelta/TimedeltaIndex of the results
-        - if False returns a np.timedelta64 or ndarray of values of dtype
-          timedelta64[ns]
+    arg : str, timedelta, list-like or Series
+          The argument which needs to be converted to timedelta.
+    unit : str, default 'ns'
+         ('Y', 'M', 'W', 'D', 'days', 'day', 'hours', hour', 'hr',
+         'h', 'm', 'minute', 'min', 'minutes', 'T', 'S', 'seconds',
+         'sec', 'second', 'ms', 'milliseconds', 'millisecond',
+         'milli', 'millis', 'L', 'us', 'microseconds', 'microsecond',
+         'micro', 'micros', 'U', 'ns', 'nanoseconds', 'nano', 'nanos',
+         'nanosecond', 'N').
+    box : bool, default True
+          If True returns a Timedelta/TimedeltaIndex of the results.
+          If False returns a np.timedelta64 or ndarray of values of dtype
```

Comment:
I think this will render better as a list, as it was before. Also, I wouldn't abbreviate numpy in the description, and we should probably use it for `numpy.ndarray`.